extensions/sql-migration/src/constants/helper.ts

```diff
@@ -150,6 +201,36 @@ export function isBlobMigration(migration: DatabaseMigration | undefined): boole
 	return migration?.properties?.backupConfiguration?.sourceLocation?.fileStorageType === FileStorageType.AzureBlob;
 }
 
+export function isShirMigration(migration?: DatabaseMigration): boolean {
+	return isLogicalMigration(migration)
+		|| isFileShareMigration(migration);
+}
+
+export function isLogicalMigration(migration?: DatabaseMigration): boolean {
+	return migration?.properties?.kind === AzureResourceKind.ORACLETOSQLDB
+		|| migration?.properties?.kind === AzureResourceKind.SQLDB;
+}
+
+export function isFileShareMigration(migration?: DatabaseMigration): boolean {
+	return migration?.properties?.backupConfiguration?.sourceLocation?.fileStorageType === FileStorageTypeCodes.FileShare;
+}
+
+export function isSqlDbTarget(migration: DatabaseMigration): boolean {
```

Comment:
nit: I would suggest renaming these three helpers slightly so that they have consistent naming, so that it's easier to find in the future.

e.g. `isSqlDbTarget`, `isSqlMiTarget`, `isSqlVmTarget`

Reply:
on second thought, maybe these helpers aren't even needed at all? either that, or `isTargetType` isn't needed... just trying to avoid multiple helpers that seemingly do the same thing, which might mean there are more references in the future that will need to be updated or may be missed if we have multiple that do the same thing

Reply:
good refactor